packages/terra-clinical-modal-manager/lib/index.js

```diff
@@ -65,5 +65,4 @@ exports.default = (0, _reactRedux.connect)(mapStateToProps, mapDispatchToProps)(
 var reducers = {
```

Comment:
The `lib` directory is now [ignored from version control](https://github.com/cerner/terra-clinical/blob/master/.gitignore#L53) and [generated via the compile script after running npm install and when publishing an npm package](https://github.com/cerner/terra-clinical/blob/master/package.json#L55-L56).

This means we can delete this file.